solidity/contracts/test/ECDSARewardsStub.sol

```diff
@@ -0,0 +1,22 @@
+pragma solidity 0.5.17;
+
+import "../../contracts/ECDSARewards.sol";
+
+/// @title ECDSA Rewards Stub for ecdsa rewards testing
+/// @dev This contract is for testing purposes only.
+contract ECDSARewardsStub is ECDSARewards {
+    constructor(
+        address _token,
+        address payable _factoryAddress,
+        address _tokenStakingAddress
+    )
+        public
+        ECDSARewards(
+            _token,
+            _factoryAddress,
+            _tokenStakingAddress
+        )
+    {
+        beneficiaryRewardCap = 10000 * 10**18;
+    }
```

Comment:
Wouldn't it be cleaner to provide a setter on the stub that would be called with a value provided by test?

Reply:
Hm, I don't know if it is cleaner, because it is a "constant" value that should probably not be changed in tests just like in a real `ECDSARewards` contract.  I believe we have a similar example in `TokenStakingStub` where we declare `minimumStake` without a setter. 
But if you think we should have a setter, I can certainly add it.

Reply:
I mean... we update this value for tests and we have a good reason for doing that. I think we should either have a setter and set the value on a stub from tests or don't have a stub at all and test against the real value. Latter is not possible with the test harness we have so I'd go with option one.

Reply:
Done